README.md

```diff
@@ -7,7 +7,13 @@ Spreadsheet in the Jupyter notebook:
    
 # Installation
 
-To install use pip:
+The easiest way to install ipysheet is with conda:
```

Comment:
easiest? I'm not sure we should have an opinion about it, currently it's for sure the slowest :)

Reply:
I can write the slowest then :P. I'll remove this comment, thanks for the review :)